src/backend/dx12/src/device.rs

```diff
@@ -1943,17 +1943,15 @@ impl d::Device<B> for Device {
             let semantics = vertex_semantic_remapping
```

Comment:
similar question here - why would we expect `vertex_semantic_remapping` to not have our attribute location?